connectors/byoc.py

```diff
@@ -700,6 +700,10 @@ async def sync(self, sync_job_index, source_klass, elastic_server, bulk_options)
             await data_provider.ping()
             await asyncio.sleep(0)
 
+            logger.debug(f"Validating configuration for {data_provider}")
+            await data_provider.validate_configuration()
+            await asyncio.sleep(0)
```

Comment:
you can remove this `await asyncio.sleep(0)`

Reply:
Removed in https://github.com/elastic/connectors-python/pull/552/commits/8b3ad71119e47b86569aa95aa03568f981c0d4eb